keccak256/src/permutation/rho_checks.rs

```diff
@@ -138,18 +136,9 @@ pub struct LaneRotateConversionConfig<F> {
 impl<F: Field> LaneRotateConversionConfig<F> {
     pub fn configure(
         meta: &mut ConstraintSystem<F>,
-        lane_idx: usize,
-        lane: Column<Advice>,
```

Comment:
We can still keep this lane and enable the copy for it.

Reply:
I'm not sure I do follow. They'll be unused inside. So I thought the best was to enable it in the top level Config that instanciates `LaneRotateConversionConfig`.

Reply:
Enabling it in the top-level Config sounds good.